hypothesis-python/docs/reference/strategies.rst

```diff
@@ -1,15 +1,216 @@
+.. _strategies:
+
+Strategies reference
+====================
+
+Primitives
+----------
```

Comment:
I think we'd really benefit from a paragraph of explanation right at the top of the reference page: what is a strategy?  the fact that you can combine them in various ways, and a link to `@given()`.